From 5f17fc66523c77fec45e6a14348638f7fdf4082c Mon Sep 17 00:00:00 2001
From: Francois Gaffie <francois.gaffie@intel.com>
Date: Wed, 18 Jul 2012 18:34:50 +0200
Subject: [PORT FROM MAIN] Add parameter key definition for HAC

BZ: 47599

This patch adds both HAC key define and HAC value pairs define.
HAC stands for Hearing Aid Compatible.

Feature: AudioComms, HAC
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: yes
Change-Id: I559643085412a11dcaff882cf72dc11ebb6ec693
Orig-Change-Id: I4e6a6a5486eaf29d26d2005bde64e9d3bd162d80
Signed-off-by: Francois Gaffie <francois.gaffie@intel.com>
Signed-off-by: Renaud Viry <renaudx.viry@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 3a0962e..0da8e59 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -90,6 +90,11 @@ __BEGIN_DECLS
 #define AUDIO_PARAMETER_VALUE_TTY_HCO "tty_hco"
 #define AUDIO_PARAMETER_VALUE_TTY_FULL "tty_full"
 
+/* HAC device selection */
+#define AUDIO_PARAMETER_KEY_HAC_SETTING "HACSetting"
+#define AUDIO_PARAMETER_VALUE_HAC_ON "ON"
+#define AUDIO_PARAMETER_VALUE_HAC_OFF "OFF"
+
 /* A2DP sink address set by framework */
 #define AUDIO_PARAMETER_A2DP_SINK_ADDRESS "a2dp_sink_address"
 
-- 
1.7.4.1

